docs/conf.py

```diff
@@ -0,0 +1,52 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Snowfakery"
+copyright = "2020, Paul Prescod and Salesforce.org Release Engineers"
```

Comment:
Doesn't Salesforce own the copyright?

Reply:
Good catch. sphinx-quickstart asks:

> Author name(s): 

Then it outputs a copyright statement!

Reply:
a812325